packages/atlas/src/utils/time.ts

```diff
@@ -31,6 +33,34 @@ export const formatDurationShort = (duration: number, showHours?: boolean): stri
   return `${minutes}:${normalize(seconds)}`
 }
 
+export const formatDurationBiggestTick = (duration: number): string => {
+  let remaining = duration
+
+  const hours = Math.floor(remaining / SECONDS_IN_HOUR)
+  remaining = remaining % SECONDS_IN_HOUR
+
+  const minutes = Math.floor(remaining / MINUTES_IN_HOUR)
+  remaining = remaining % MINUTES_IN_HOUR
+
+  const seconds = remaining
+
+  if (hours > 24) {
```

Comment:
I think it should be `hours > 23` as e.g 24 hours 30 minutes should be displayed as 1 day already